x-pack/plugins/endpoint/public/embeddables/resolver/embeddable.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EmbeddableInput,
+  IContainer,
+  Embeddable,
+} from '../../../../../../src/plugins/embeddable/public';
+
+export class ResolverEmbeddable extends Embeddable {
+  public readonly type = 'resolver';
```

Comment:
You probably want to have a single const and export that so a caller can get the factory using it,

```
context.embeddable.getEmbeddableFactory(RESOLVER_EMBEDDABLE).create()
```